interaction-manager/src/lib_components/spatial-navigation.js

```diff
@@ -426,12 +507,23 @@ class SpatialNavigation {
     focusLast(area = 'default') {
         if (!this.enabled) return;
 
-        const navigatableElements = this.navigatableElements[area];
+        const navigatableElements = this.navigatableElements[area].elements;
         if (!navigatableElements || navigatableElements.length === 0) {
             return console.error(`The area '${area}' you are trying to focus doesn't exist or the spatial navigation hasn't been initialized`);
         }
 
-        this.lastFocusedElement = navigatableElements.slice(-1)[0];
+        let element;
+        for (let i = navigatableElements.length - 1; i >= 0; i--) {
+            if (!navigatableElements[i].hasAttribute('disabled')) {
+                element = navigatableElements[i];
+                break;
+            }
+        }

```

Comment:
There is `findLast` method but we do not support it 😢 https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/findLast

Reply:
Maybe move this in a method for finding the last focused element